src/java/org/apache/cassandra/audit/AuditLogOptions.java

```diff
@@ -40,8 +47,220 @@
     /**
      * AuditLogs directory can be configured using `cassandra.logdir.audit` or default is set to `cassandra.logdir` + /audit/
      */
-    public String audit_logs_dir = System.getProperty("cassandra.logdir.audit",
-                                                      System.getProperty("cassandra.logdir",".")+"/audit/");
+    public String audit_logs_dir = Paths.get(System.getProperty("cassandra.logdir.audit",
```

Comment:
we shouldn't be specifying property keys directly anymore. please use `CassandraRelevantProperties`

Reply:
good catch, fixed.